Source/Core/DolphinQt/DolphinQt.vcxproj

```diff
@@ -426,6 +426,7 @@
     </ProjectReference>
   </ItemGroup>
   <Import Project="$(ExternalsDir)bzip2\exports.props" />
+  <Import Project="$(ExternalsDir)curl\exports.props" />
```

Comment:
What do these additions have to do with -Wdeprecated-declarations? They make a bit of sense to me (though I'm not sure whether they're needed or not for common-only stuff), but it seems odd still since the vcxproj file isn't used by GCC.

Reply:
The newer progress callback takes `curl_off_t` arguments rather than floats.  Originally, the \<curl/curl.h\> header was only included in `/Core/Common/HttpRequest.cpp`.  Including it in `/Core/Common/HttpRequest.h` to access the `curl_off_t` typedef manifested into a build error due to certain parts of the Windows build system lacking the curl external.